packages/vuetify/src/components/VFileInput/VFileInput.tsx

```diff
@@ -193,6 +195,7 @@ export const VFileInput = genericComponent<VFileInputSlots>()({
                 disabled={ isDisabled.value }
                 focused={ isFocused.value }
                 error={ isValid.value === false }
+                style={ !hasInput ? { display: 'none' } : props.style }
```

Comment:
consider to use existing utility class `d-none`

Reply:
Utility classes should not be used in components.